06week/fizzbuzz/index.js

```diff
@@ -0,0 +1,20 @@
+//Fizzbuzz
+// Using the coding area below, looping through all numbers 1 through 1000,
+// I want you to log:
+//
+// 'Fizz' if the number is divisible by only three
+// 'Buzz' if the number is divisible by only five
+// 'FizzBuzz' if the number is divisible by three and five
+// the number itself if it is none of these
+
+
+for (x = 0; x < 1001; x++) {
+  if (x % 3 === 0)
+  console.log('Fizz');
+  if else (x % 5 === 0)
+  console.log('Buzz');
+  else ((x % 3 === 0) && (x % 3 === 0))
```

Comment:
this should be x%5 fro the second check.
Also, this will not work since if it is divisable by 3, it would be caught in the first if condition